nibabel/nifti1.py

```diff
@@ -1911,7 +1911,7 @@ def _chk_offset(hdr, fix=False):
             return hdr, rep
         if magic == hdr.single_magic and offset < hdr.single_vox_offset:
             rep.problem_level = 40
-            rep.problem_msg = 'vox offset %d too low for single file nifti1' % offset
+            rep.problem_msg = f'vox offset {offset} too low for single file nifti1'
```

Comment:
I think that's the reason for the current CI error. `offset` is actually a float, but it was printed as in int:
```
  E         - vox offset 10 too low for single file nifti1; setting to minimum value of 352
  E         + vox offset 10.0 too low for single file nifti1; setting to minimum value of 352
```